using-mobile-extensions/adobe-target/target-api-reference.md

```diff
@@ -204,173 +281,331 @@ ACPTarget.getTntId({tntId in
+let targetParameters = ACPTargetParameters(parameters: nil, profileParameters: nil, product: product, order: order)
+
+ACPTarget.locationClicked(withName: "cartLocation", targetParameters: targetParameters)
 ```
 {% endtab %}
 
 {% tab title="React Native" %}
-### resetExperience
+
+#### Syntax
+
+```javascript
+locationClickedWithName(name: string, parameters?: ACPTargetParameters)
+```
+
+* _name_ is a string containing mbox location for which click notification is to be sent to Target.
+* _parameters_ is the configured `ACPTargetParameters` for the request.
+
+#### Examples
 
 **JavaScript**
 
 ```javascript
-ACPTarget.resetExperience();
+// Mbox parameters
+var mboxParameters = {"membership": "prime"};
+
+// Product parameters
+var productParameters = new ACPTargetProduct("CEDFJC", "Electronics");
+
+// Order parameters
+var orderParameters = new ACPTargetOrder("NJJICK", 650, ["81","123","190"]);
+
+// Profile parameters
+var profileParameters = {"ageGroup": "20-32"};
+
+// Create Target parameters
+var product = new ACPTargetProduct("24D334", "Stationary");
+var order = new ACPTargetOrder("ADCKKBC", 400.50, ["34","125"]);
+var targetParameters = new ACPTargetParameters(null, null, product, order);
+
+ACPTarget.locationClickedWithName("cartLocation", targetParameters);
 ```
+
 {% endtab %}
+
 {% endtabs %}
 
-## Set preview restart deeplink
+## locationsDisplayed
 
-Use this API to set the Target preview URL to be displayed when the preview mode is enabled and preview selections are confirmed.
+Use this API to send a location \(mbox\) display notification to configured Target server. This API should be invoked for a prefetched mbox, after the mbox content is retrieved using [retrieveLocationContent API](https://aep-sdks.gitbook.io/docs/using-mobile-extensions/adobe-target/target-api-reference#retrieve-location-content-requests). If no previous prefetch request is made and the mbox content is retrieved using `retrieveLocationContent` API, then calling this API does not trigger a notification request to the Target server.
+
+{% hint style="warning" %}
+Do not use this API with the deprecated [loadRequests](https://aep-sdks.gitbook.io/docs/using-mobile-extensions/adobe-target/target-api-reference-deprecated#load-target-requests). For a prefetched mbox, on calling the deprecated `loadRequests` API, mbox display notification is internally sent to the Target server by the SDK.
+{% endhint %}
 
 {% tabs %}
 {% tab title="Android" %}
-### setPreviewRestartDeeplink
 
 #### Syntax
 
-```text
-public static void setPreviewRestartDeepLink(final Uri deepLink)
+```java
+public static void locationsDisplayed(final List<String> mboxNames, final TargetParameters targetParameters)
 ```
 
+* _mboxNames_ is a list of mbox locations for which display notification is to be sent to Target.
+* _targetParameters_ is the configured `TargetParameters` for the request.
+
```

Comment:
Please reword the bullet points as follows:
"* *mboxNames* is a list of the mbox locations for which the display notification will be sent to Target.
* *targetParameters* is the configured `TargetParameters` for the request."